Adds landscape layout for PlayerActivity

amilcar-andrade opened the pull request:
## Description
Creates activity_dribbble_player_details.xml to encapsulate all
of the details about the player

Creates layout-w480dp folder which contains a landscape layout for
activity_dribbble_player_details

## After
![16 - 1](https://cloud.githubusercontent.com/assets/802308/19949381/a7aff836-a127-11e6-9671-f8eaa83b2a32.png)

## Before
![16 - 2](https://cloud.githubusercontent.com/assets/802308/19949382/a8cc1c4a-a127-11e6-9c3d-7aa1d3c6f6c8.png)

@nickbutcher I tried to improve the landscape mode of `dribble player` layout. Let me know if you think is a good idea or not.

Ekt0s commented:
I'm not really fan of the empty view on the bottom, otherwise it has a little bit more information push up directly readable that's cool :D

amilcar-andrade commented:
@Ekt0s There is no empty view. It is an image that is still loading.

nickbutcher commented:
Great idea, thanks for the change!

I'm not completely sold on this layout. WDYT about this?

![player-landscape](https://cloud.githubusercontent.com/assets/352556/20063803/6bfaf3c2-a500-11e6-9401-b46bfd7dc906.png)

nickbutcher commented:
p.s. you should follow @romannurik 😉

romannurik commented:
😁

amilcar-andrade commented:
hehe, I followed him in all of the social media platforms and dribbble. He has a lot of good projects that really helped me in the past and present. Thanks @romannurik !!!
* https://romannurik.github.io/AndroidIconAnimator/
* https://github.com/romannurik/LayerVisualizer
* https://romannurik.github.io/AndroidAssetStudio/

But, when I took the screenshot when I was doing development. So I tried to not be logged in.

amilcar-andrade commented:
@nickbutcher sure, I am going to try your suggestion. Thanks!

Ekt0s commented:
Oh okay @amilcar-andrade my bad ;)

amilcar-andrade commented:
@Ekt0s No worries brawh!

amilcar-andrade commented on app/src/main/res/layout/activity_dribbble_player.xml:
```diff
@@ -36,120 +36,7 @@
-            android:paddingTop="@dimen/padding_large"
-            android:paddingBottom="@dimen/padding_large"
-            android:drawableTop="@drawable/avd_likes"
-            android:background="@null"
```
@nickbutcher I removed this line `android:background="@null"`

amilcar-andrade replied:
Got it. Thanks for letting me know

amilcar-andrade commented:
@nickbutcher I implemented your suggestion. Let me know what do you think. Thanks!
![screenshot_20161107-171603](https://cloud.githubusercontent.com/assets/802308/20080018/84c58330-a515-11e6-803c-f073733ddbff.png)

nickbutcher commented on app/src/main/res/layout-w480dp/activity_dribbble_player_details.xml:
```diff
@@ -0,0 +1,115 @@
+<?xml version="1.0" encoding="utf-8"?>
```
So right now the avatar image is left aligned and the bio etc is to the right of that. I wonder how this will scale with larger devices e.g. tablets in landscape.

An alternative approach might be to use a vertical guideline (perhaps at 33% from left) and align content to that i.e. the avatar right-aligns to the guideline and the bio etc left-aligns to it. That way on wider devices the bio text space doesn't get overly large. WDYT?

amilcar-andrade replied:
You are right, this should only apply for landscape mode. And I will try your other suggestion.

nickbutcher commented on app/src/main/res/values-w480dp/dimens.xml:
```diff
@@ -20,4 +20,6 @@
     <dimen name="bottom_sheet_width">480dp</dimen>
 
+    <!-- dribbble player -->
+    <dimen name="avatar_margin_start">24dp</dimen>
```
If we change the qualifiers of the layout, this will also need to move. You might also be able to use `@dimen/padding_large` here?

amilcar-andrade commented on app/src/main/res/layout/activity_dribbble_player.xml:
```diff
@@ -36,120 +36,7 @@
-            style="@style/Widget.Plaid.InlineActionButton" />
-
-    </android.support.constraint.ConstraintLayout>
+    <include layout="@layout/activity_dribbble_player_details" />
```
I did it because I did not want to also add the parent view, loading view, recycler view and the background view to the landscape version because those views are not changing. So I extracted the details version into a separate layout. But you are right, this include is only going to be used here, so I will just add those views to the landscape version. Thanks!

amilcar-andrade commented:
@nickbutcher I was addressing your comments and I tried it on a tablet on smaller phones and I think the recommended UI will not work because sometimes the name gets cut off or overlaps the actions (shots, likes, followers) depending on the length of the name. (I know I could try to truncate it but It also looks bad).  I tried a different alternative let me know what do you think. I can close the PR if you don't like any of the options. 

**Phone proposed UI**

![screenshot_20161109-182335](https://cloud.githubusercontent.com/assets/802308/20159362/258bd9b0-a6ae-11e6-88dd-b4e089e9af90.png)

![screenshot_20161109-184601](https://cloud.githubusercontent.com/assets/802308/20159344/0e134d04-a6ae-11e6-925d-8e006344ae7f.png)

**Tablet proposed UI**
![screenshot_20161109-184013](https://cloud.githubusercontent.com/assets/802308/20159282/a7d760c0-a6ad-11e6-86e9-d5ea3afbbdbb.png)

![screenshot_20161109-183948](https://cloud.githubusercontent.com/assets/802308/20159302/c835c5be-a6ad-11e6-898e-2a937e3a711a.png)


**Phone alternative UI**

![screenshot_20161109-183601](https://cloud.githubusercontent.com/assets/802308/20159337/02a111cc-a6ae-11e6-9e7f-e6010b34137c.png)

![screenshot_20161109-183438](https://cloud.githubusercontent.com/assets/802308/20159359/240997c6-a6ae-11e6-9cc1-67491fe53a95.png)


**Tablet alternative UI**

![screenshot_20161109-183815](https://cloud.githubusercontent.com/assets/802308/20159265/94e797a0-a6ad-11e6-9cda-13ea1432e81f.png)

![screenshot_20161109-183848](https://cloud.githubusercontent.com/assets/802308/20159296/b9a15dc4-a6ad-11e6-8040-4a8cc7c2a1a5.png)

amilcar-andrade commented:
@nickbutcher your layout is way better in landscape for most of the cases. So I am closing this PR
